### PR TITLE
Use MPL in RichardsFlow

### PR DESCRIPTION
**Pull request by TomFischer**
1. [ ] Feature description was added to the [changelog](https://github.com/ufz/ogs/wiki/Release-notes-6.3.0)
2. [x] Tests covering your feature were added?

- Disabling for now all "curve" ctests until Curve type MPL property is implemented.
 @TomFischer is working on it. (https://github.com/ufz/ogs/issues/2852)
- A new test for fully saturated flow was added.

3. [ ] Any new feature or behavior change was documented?

### CODE REVIEW

**Comment by endJunction** [comment]
@wenqing Could you have a look on the code changes and the project files?
We will rebase/cleanup the commits, but the end result is as of now.

File: ProcessLib/RichardsFlow/RichardsFlowFEM.h [hunk 1]
```diff
@@ -151,63 +155,87 @@ class LocalAssemblerData : public RichardsFlowLocalAssemblerInterface
...
+                        .value(vars, pos, t, dt));
+
+            auto const porosity =
+                solid_phase.property(MaterialPropertyLib::PropertyType::porosity)
+                    .template value<double>(vars, pos, t, dt);
 
-            double const Sw = _process_data.material->getSaturation(
-                material_id, t, pos, p_int_pt, temperature, pc_int_pt);
+            vars[static_cast<int>(
+                MaterialPropertyLib::Variable::capillary_pressure)] = -p_int_pt;
```
Comment (wenqing) [inline ProcessLib/RichardsFlow/RichardsFlowFEM.h]: Please move these two lines to the line after line 177, where `MaterialPropertyLib::Variable::phase_pressure` is set.
```suggestion
            vars[static_cast<int>(
                MaterialPropertyLib::Variable::capillary_pressure)] = -p_int_pt;
```

File: Tests/Data/Parabolic/Richards/flow_fully_saturated.prj [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+                        <property>
+                            <name>porosity</name>
+                            <type>Constant</type>
+                            <value>0.8</value>
+                        </property>
+                        <property>
+                            <!-- (alpha - porosity) / K_SR -->
+                            <name>storage</name>
+                            <type>Constant</type>
+                            <value>0.2e-100</value>
```
Comment (wenqing) [inline Tests/Data/Parabolic/Richards/flow_fully_saturated.prj]: So small storage, you can set it as 0.

File: ProcessLib/RichardsFlow/Tests.cmake [hunk 3]
```diff
@@ -74,19 +74,32 @@ AddTest(
...
+    EXECUTABLE ogs
+    EXECUTABLE_ARGS flow_fully_saturated.prj
+    WRAPPER time
     TESTER vtkdiff
-    REQUIREMENTS OGS_USE_MPI
-    RUNTIME 220
+    REQUIREMENTS NOT OGS_USE_MPI
     DIFF_DATA
-    ref_t_1600.000000.vtu richards_pcs_0_ts_803_t_1600_000000_0.vtu pressure pressure 1e-8 1e-3
+    GLOB flow_fully_saturated_pcs_0_ts_*.vtu pressure pressure 1e-9 0
```
Comment (wenqing) [inline ProcessLib/RichardsFlow/Tests.cmake]: Is this benchmark the same as one of LiquidFLow benchmarks? If it is, please highlight it somewhere, e.g. in the file name or reference variable name.

Comment (endJunction) [reply ProcessLib/RichardsFlow/Tests.cmake]: It is same as in RichardsMechanics, which was created based on a RichardsFlow ctest...

**Pull request merged**